.github/workflows/push-trigger.yml

```diff
@@ -22,17 +22,15 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-
-      - name: Set up Swift
-        uses: swift-actions/setup-swift@v1
-        with:
-          swift-version: '5.9.2'  
+      
+      - name: Set up Xcode
+        run: sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
 
       - name: Build project
-        run: swift build
+        run: xcodebuild -scheme VCIClient -destination 'generic/platform=iOS' build
 
       - name: Run tests
-        run: swift test
+        run: xcodebuild test -scheme VCIClientTests -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.0'
```

Comment:
Can we use generic simulator for tests, as it might be dependent on pipeline availability...
We can try something like this ->
`xcodebuild test -scheme YourAppScheme -destination 'platform=iOS Simulator,OS=latest`
'